sdk/openai/openai/review/openai-types.api.md

```diff
@@ -260,7 +260,7 @@ export interface ContentFilterResultOutput {
 
 // @public
 export interface ContentFilterResultsForChoiceOutput {
-    custom_blocklists?: Array<ContentFilterBlocklistIdResultOutput>;
+    custom_blocklists?: ContentFilterBlocklistIdResultOutput;
```

Comment:
Is this change intentional? The name of the property suggests that it is an array but this change makes it a single item, why is that?

Reply:
This change is intentional based on the TypeSpec description [here](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/OpenAI.Inference/models/completions/azure_content_filtering.tsp#L96-L103). The change happens from the `2024-04-01-preview` version